tests/providers/google/cloud/operators/test_cloud_run.py

```diff
@@ -387,3 +393,27 @@ def test_execute_with_invalid_limit(self, hook_mock):
         limit = -1
         with pytest.raises(expected_exception=AirflowException):
             CloudRunListJobsOperator(task_id=TASK_ID, project_id=PROJECT_ID, region=REGION, limit=limit)
+
+
+class TestCloudRunCreateServiceOperator:
+    def test_template_fields(self):
+        operator = CloudRunCreateServiceOperator(
+            task_id=TASK_ID, project_id=PROJECT_ID, region=REGION, service_name=SERVICE_NAME, service=SERVICE
+        )
+
+        _assert_common_template_fields(operator.template_fields)
+        assert "service_name" in operator.template_fields
+
+    @mock.patch(CLOUD_RUN_SERVICE_HOOK_PATH)
+    def test_create(self, hook_mock):
+        hook_mock.return_value.create_service.return_value = SERVICE
+
+        operator = CloudRunCreateServiceOperator(
+            task_id=TASK_ID, project_id=PROJECT_ID, region=REGION, service_name=SERVICE_NAME, service=SERVICE
+        )
+
+        operator.execute(context=mock.MagicMock())
+
+        hook_mock.return_value.create_service.assert_called_once_with(
+            service_name=SERVICE_NAME, region=REGION, project_id=PROJECT_ID, service=SERVICE
+        )
```

Comment:
Can you please also add test for DeleteOperator?

Reply:
done.